backend/app/store/engine/bolt.go

```diff
@@ -205,6 +223,16 @@ func (b *BoltDB) Flag(req FlagRequest) (val bool, err error) {
 	return b.setFlag(req)
 }
 
+// UserDetail sets and gets detail values
+func (b *BoltDB) UserDetail(req UserDetailRequest) (val string, err error) {
+	if req.Update == "" { // read detail value, no update requested
```

Comment:
How should I distinguish read requests from entry deletion requests? Flags have FlagNonSet zero value, by the same logic I could add some special string value for deletion ("delete"?) but it doesn't feel right.